docs/requirements.txt

```diff
@@ -0,0 +1 @@
+furo
```

Comment:
> this change would require changing the readthedocs configuration to install the docs/requirements.txt packages

Is such new dependency strictly necessary? Our docs on RTD was mean to be a no-frills rendering, because we tried to minimize our effort on maintenance.

Currently, that new [`furo` module happens to requires Python 3.5+](https://pypi.org/project/furo/), and RTD's default build interpretor happens to be Python 2.7 (who would know that...), so [it failed](https://readthedocs.org/projects/msal-python/builds/13156354/). Then I tried switching the RTD interpretor into Python 3.x, but then somehow it [failed with other error](https://readthedocs.org/projects/msal-python/builds/13156392/).

FWIW, Microsoft Docs team is also planning to generate and host MSAL docs on docs.microsoft.com. Perhaps there would be a unified scheme there.

Reply:
Fixed in next commit I hope by including the parent directory so it will install the requirements of the package